models/gc_picai_baseline/config/default.yml

```diff
@@ -0,0 +1,35 @@
+general:
+  data_base_dir: /app/data
+  version: 1.0
+  description: Prostate MRI classification default (dicom to json)
+
+execute:
+- FileStructureImporter
+- MhaConverter
+- PicaiBaselineRunner
+- ReportExporter
+- DataOrganizer
+
+modules:
+  FileStructureImporter:
+    input_dir: input_data
+    structures:
+      - $sid@instance/images/transverse-adc-prostate-mri@dicom:mod=mradc
+      - $sid/images/transverse-t2-prostate-mri@dicom:mod=mrt2
+      - $sid/images/transverse-hbv-prostate-mri@dicom:mod=mrhbv
```

Comment:
Agree that this cannot be extracted trustfully from DICOM.

Some comments:
- the `images` subfolder is not required and should be removed.
- `mod=mr` must be set for all data, which is modality MR.
- you can introduce a model-specific meta key to distinguish between the three models.
- The sub-folder names contain redundant information that might be removed (`...-prostate-mri`)
- You can utilize the subfodler name (where the dicom is contained) to generate a meta key from that folder, e.g., `$sid/$type@dicom:mod=mr`

Reply:
Okay, I agree with all the comments and it can be easily simplified. This folder structure reflects the grand-challenge input interface hence I kept it. Modality should be mr only and the type should become some other tag. I will change it to something else as suggested.